html/NedapApiWebSocketClient.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Nedap Renos WebSocket Test Client</title>
+    <script language="javascript" type="text/javascript">
+        var CLOSING = 2;
+        var CLOSED = 3;
+        var RENOS_EVENTS_URI = "ws://192.168.133.1/api/v2/events";
```

Comment:
Please add a textbox on the page to enter an IP address. This example suggests it's ok to use the API via the USB service port. And we don't want that.